--- Parse-RecDescent-1.77/find-perl-requires	Thu Jan  1 01:00:00 1970
+++ Parse-RecDescent-1.77.dep/find-perl-requires	Sat Feb 19 22:50:19 2000
@@ -0,0 +1,11 @@
+#!/bin/sh
+ulimit -c 0
+
+filelist=`sed "s/['\"]/\\\&/g"`
+
+requires="`echo $filelist|/usr/lib/rpm/find-requires|grep -v "no package provides"`"
+requires_perl="`/usr/lib/rpm/perl.req $filelist| grep -v "no package provides" |grep -v 'perl(Calc)'`"
+requires_mod="`rpm -q --whatprovides --qf "%{NAME}\n"  $requires_perl 2>/dev/null|grep -v "no package provides"`"
+echo "$requires
+$requires_perl
+$requires_mod"|  sort -u
